### ISSUE TITLE
Feature request: Navigate from "Rules Used in Current Selection" to the rule's options

### ISSUE THREAD

**Issue by ConjuringCoffee** [SAP/abap-cleaner#161]
Hi Jörg-Michael, the interactive clean-up shows the "Rules Used in Current Selection". I think it would be useful if it was possible to navigate directly to options of one of these rules. This would be especially useful when initially setting up a custom profile or when investigating unwanted / unexpected behavior.

I think an intuitive way to do this would be if the "Configure" button would open up to the rule last clicked on in the "Rules Used in Current Selection" section. What do you think?

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

good idea! I think we could even consider a double-click for this (as a bow to SAP GUI-like navigation ;-) or Strg+Click (more ADT-like), or select + F3 or …

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

thanks again for this nice idea, it's really a usability improvement! It will be available with the next release:
* after selecting a rule – with or without (un)checking it – this rule will be shown when press "Configure…" the next time; 
* same for (un)checking the rule via keyboard (pressing a, b, c, …);
* Ctrl + click on a rule in the list will take you directly to the configuration of the rule (F3 is already used for text search).
 
I tried to figure out how to underline the list entry while Ctrl is pressed and the mouse hovers over the rule list, but that seemed rather complicated with this SWT control. Anyway, I guess if someone presses Ctrl, they already know what they're doing, and I added a line on this in the documentation.

Kind regards,
Jörg-Michael
